Gems/ScriptCanvas/Code/Editor/View/Widgets/NodePalette/GeneralNodePaletteTreeItemTypes.h

```diff
@@ -71,9 +71,11 @@ namespace ScriptCanvasEditor
 
             const char* className = m_className.toUtf8().data();
             auto behaviorClass = behaviorContext->m_classes.find(className);
-
-            ScriptCanvasEditorTools::TranslationGeneration translation;
-            translation.TranslateBehaviorClass(behaviorClass->second);
+            if (behaviorClass != behaviorContext->m_classes.end())
```

Comment:
You can use the new [contains](https://en.cppreference.com/w/cpp/container/unordered_map/contains) hotness to reduce this to a single statement